packages/web-components/src/components/content-item-horizontal/__stories__/README.stories.mdx

```diff
@@ -18,43 +18,65 @@ Here's a quick example to get you started.
 import '@carbon/ibmdotcom-web-components/es/components/cta/link-list-item-cta';
+import '@carbon/ibmdotcom-web-components/es/components/video-player/video-player-container';
+import '@carbon/ibmdotcom-web-components/es/components/image/image';
 ```
 
 ### HTML
 
 ```html
-  <dds-content-item-horizontal>
-    <dds-content-item-horizontal-eyebrow>Lorem ipsum</dds-content-item-horizontal-eyebrow>
-    <dds-content-item-heading>Aliquam condimentum</dds-content-item-heading>
-    <dds-content-item-horizontal-copy>Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin.</dds-content-item-horizontal-copy>
-    <dds-link-list slot="footer" type="vertical">
-      <dds-link-list-item-cta
-        icon-placement="right"
-        href="https://www.ibm.com"
-        cta-type="local"
-      >
-        Learn more
-      </dds-link-list-item-cta>
-      <dds-link-list-item-cta
-        icon-placement="right"
-        href="https://www.ibm.com"
-        cta-type="external"
-      >
-        Microservices and containers
-      </dds-link-list-item-cta>
-    </dds-link-list>
-  </dds-content-item-horizontal>
+<dds-content-item-horizontal>
+  <dds-content-item-horizontal-eyebrow>Lorem ipsum</dds-content-item-horizontal-eyebrow>
+  <dds-content-item-heading>Aliquam condimentum</dds-content-item-heading>
+  <dds-content-item-horizontal-copy
+    >Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit.
+    Phasellus at elit sollicitudin.</dds-content-item-horizontal-copy
+  >
+  <dds-link-list slot="footer" type="vertical">
+    <dds-link-list-item-cta icon-placement="right" href="https://www.ibm.com" cta-type="local">
+      Learn more
+    </dds-link-list-item-cta>
+    <dds-link-list-item-cta icon-placement="right" href="https://www.ibm.com" cta-type="external">
+      Microservices and containers
+    </dds-link-list-item-cta>
+  </dds-link-list>
+</dds-content-item-horizontal>
+```
+
+#### With Media
+
+```html
+<dds-content-item-horizontal-media align="right">
+  <dds-image
+    slot="media"
+    alt="Lorem ipsum"
+    default-src="https://fpoimg.com/672x336?text=2:1&bg_color=ee5396&text_color=161616"
+  ></dds-image>
+  <dds-content-item-heading>Aliquam condimentum</dds-content-item-heading>
+  <dds-content-item-horizontal-copy
+    >Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit.
+    Phasellus at elit sollicitudin.</dds-content-item-horizontal-copy
+  >
+  <dds-link-list slot="footer" type="vertical">
+    <dds-link-list-item-cta icon-placement="right" href="https://www.ibm.com" cta-type="local">
+      Learn more
+    </dds-link-list-item-cta>
+    <dds-link-list-item-cta icon-placement="right" href="https://www.ibm.com" cta-type="external">
+      Microservices and containers
+    </dds-link-list-item-cta>
```

Comment:
In the example there's only one CTA, perhaps can remove in the Docs as well? I don't recall if it was the intent to have more than one CTA here.

Reply:
Taking this back, I saw there's more than one CTA in the designs!